extract-method-core/src/main/java/org/jetbrains/research/extractMethod/core/extractors/NegativesExtractor.java

```diff
@@ -51,7 +51,7 @@ public static String statementsAsStr(List<PsiStatement> statementList) {
 
     private static String getRelativePath(Project project, PsiJavaFile file) {
         // Removes prefix "PsiDirectory:" (13 characters)
```

Comment:
please, remove a comment. BTW, have you tested this method? does it work as we expect?

Reply:
yes, tested and it works like a charm, thanks for a suggestion!